src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java

```diff
@@ -49,6 +49,7 @@
                                                                                                               "sepadirectdebit");
     // Period is a bit aggressive by default. SOFORT (directEbanking) payment can take up to 14 days.
     public static final String DEFAULT_OFFLINE_BANK_TRANSFER_PENDING_PAYMENT_EXPIRATION_PERIOD = "P7d";
+    public static final String FALL_BACK_MERCHANT_ACCOUNT_KEY = "FALLBACK";
```

Comment:
Could you update the `README` and document this feature?

Reply:
Done in e80ba9b